docs/install.md

```diff
@@ -106,10 +106,10 @@ the [*Cray System Management Documentation*](https://cray-hpe.github.io/docs-csm
 The following is the procedure to globally configure the username used by SAT and
 authenticate to the API gateway.
 
-1. Generate a default SAT configuration file, if one does not exist.
+1. (`ncn-m001#`) Generate a default SAT configuration file, if one does not exist.
 
    ```screen
-   ncn-m001# sat init
+   sat init
    Configuration file "/root/.config/sat/sat.toml" generated.
```

Comment:
Here's another example where removing the command prompt from our code blocks makes it unclear which lines are the command(s) the user needs to run and which are the example output. In order to remove the command prompts from the code blocks, we need to also separate the command from the command output. Something like:

````
   ```sh
   sat init
   ```

   The output from this command should look something like:

   ```screen
   Configuration file "/root/.config/sat/sat.toml" generated.
   ```
````

Reply:
Updated in next commit.